packages/apps/plugins/plugin-debug/src/DebugPlugin.tsx

```diff
@@ -105,6 +141,8 @@ export const DebugPlugin = (): PluginDefinition<DebugPluginProvides> => {
           case 'main': {
             if (isDebug(data)) {
               return DebugMain;
+            } else if (data === 'dxos-devtools') {
```

Comment:
just 'devtools' 
perhaps also remove `isDebug` if the `data` attribute is all that is required?
and shouldn't that be role?
i'm still confused about detecting this.